docs/CONNECTOR_PROTOCOL.md

```diff
@@ -19,6 +19,30 @@ Connectors can:
 
 At this stage, our assumption is that one connector will manage one index, and one index will have only one connector associated with it. This may change in the future.
 
+### Data freshness
+
+Data from remote sources are synced via [sync jobs](DEVELOPING.md#syncing), which means there can be data discrepancy between the remote source and Elasticsearch until the next sync job runs.
+
+There are some data sources with streaming option, which allows external systems to capture changes in real-time. This option has been evaluated, and will not in our roadmap in the near future.
```

Comment:
It's probably good to have a list of these data sources somewhere and reference it from this document. Do you know if we already have one?

Reply:
Streaming API for MongoDB, and change notifications for Sharepoint and OneDrive. I add add these two as example and link to the reference doc.

Reply:
Added in https://github.com/elastic/connectors-python/pull/1428/commits/decac08da3d18537d749cdd2fbece5c7a7933973

Reply:
Removed OneDrive as it's in the connector catalog yet: https://github.com/elastic/connectors-python/pull/1428/commits/fa9bcdc7997c571f7116ad58a29b5b1ea9fb1169